netflix-roulette/src/pages/home/components/Catalog/components/MoviesList/MovieCard/MovieCard.tsx

```diff
@@ -37,6 +43,7 @@ export const MovieCard = ({
           ((e.target as HTMLImageElement).src =
```

Comment:
Doesn't seem like a good idea (direct DOM manipulations). I would rather recommend to implement that behavior using  appropriate state variable.